client/app/app.js

```diff
@@ -70,6 +70,10 @@ angular.module("app", [
 .run(AppLocalizeRun)
 .config(AppMaterial)
 .config(AppHTTPErrorHandler)
+.config(( $mdGestureProvider ) => {
```

Comment:
Is there documentation on this? I found https://github.com/angular/code.material.angularjs.org/blob/master/HEAD/partials/api/material.core.gestures/service/$mdGestureProvider.html

I guess the touch event is intercepted by $mdGesture and does not reach leaflet. But is there another event emitted that could be used by leaflet?